packages/nuxt/src/core/builder.ts

```diff
@@ -18,6 +18,11 @@ export async function build (nuxt: Nuxt) {
 
   if (nuxt.options.dev) {
     watch(nuxt)
+
+    process.on('exit', () => {
+      nuxt.callHook('close', nuxt)
+    })
```

Comment:
this doesn't seem to result in this hook being called:

```ts
export default defineNuxtConfig({
  hooks: {
    close: () => {
      console.log('closing things')
    },
  },
})
```

Reply:
oh indeed, it's weird, the exit listener is correctly called

Reply:
hmmm the main issue is that callHook is async

Reply:
tbh it is probably not possible at least, we can't support it on windows since SIGTERM directly terminate the process without any way to handle the exit. On linux tho, we can probably try the `beforeExit` hook.

WDYT ? Should we drop this ?